packages/react-components/src/entities/EventSidebar/details/Measurements.js

```diff
@@ -9,18 +9,23 @@ export function Measurements({ data }) {
     const [fixedColumn, setFixed] = useState(true);
 
     let hasMeasurements = false;
-    if (data.results.documents.results
-        && data.results.documents.results.length > 0
-        && data.results.documents.results[0].measurementOrFacts
-        && data.results.documents.results[0].measurementOrFacts.length > 0) {
+    if (data.documents.results
+        && data.documents.results.length > 0
+        && data.documents.results[0].measurementOrFacts
+        && data.documents.results[0].measurementOrFacts.length > 0) {
         hasMeasurements = true;
     }
 
     if (!hasMeasurements){
         return <></>
     }
 
-    const results = data.results.documents.results[0].measurementOrFacts;
+    //const results = data.documents.results[0].measurementOrFacts;
+    let results = data.documents.results.reduce(function(measurements, result){
+        measurements.push(result.measurementOrFacts);
+        return measurements;
+    },[]);
+    results = results.flat();
```

Comment:
Reassigning a constant. I'm surprised if this doesn't throw a warning (it will work, but please change it to `let` or use a new variable)

Reply:
Fixed